lorax/lorax.py

```diff
@@ -102,18 +419,28 @@ def __init__(self, clf, test_mat, id_col=None,
             self.X_test["Intercept"] = [self.clf.intercept_[0] for i in range(len(self.X_test))]
 
         # pre-calcuate feature distribution statistics for each feature
-        self._populate_feature_stats()
+        self.feature_stats = self.populate_feature_stats(test_mat=self.X_test)
+   
+    # TODO: make protected again. Making public for testing
```

Comment:
should probably go ahead and do this here, unless there's a reason to keep it public?